### DISCUSSION TITLE
Plan for python support?

### DISCUSSION THREAD

**Discussion by wzjin2017** [openfheorg/openfhe-development#332] [Ideas]
Hi developers,

Do you have a plan for adding features like python wrappers to write OpenFHE application codes in python? For example, processing tensors.

**Comment by IanQS** [comment]
Hi there! Just so you know, we tend to be more responsive on our [discourse forum](https://openfhe.discourse.group/), but know that we are hoping to release a python wrapper soon. See the progress at [openfhe-python](https://github.com/openfheorg/openfhe-python)
